### PR TITLE
Adding a new movie name and a book name

### PR DESCRIPTION
**Pull request by HasnaatCodes**

### CODE REVIEW

File: books.md [hunk 1]
```diff
@@ -8,4 +8,5 @@
 - The ocean at the end of the lane
 - Smoke and mirrors
 - Neverwhere
-- American Gods
\ No newline at end of file
+- American Gods
+- Sandman
```
Comment (StuartCowley) [inline books.md]:
```suggestion
- Sand Man
```

File: movies.md [hunk 2]
```diff
@@ -10,4 +10,5 @@
 - Red Dragon
 - Fight Club
 - The Italian Job
-- Bourne Legacy
\ No newline at end of file
+- Bourne Legacy
```
Comment (StuartCowley) [inline movies.md]:
```suggestion
```

File: books.md [hunk 3]
```diff
@@ -8,4 +8,5 @@
 - The ocean at the end of the lane
 - Smoke and mirrors
 - Neverwhere
-- American Gods
\ No newline at end of file
+- American Gods
```
Comment (StuartCowley) [inline books.md:11]: Should 'Gods' be not capitalised?

**Review by StuartCowley — CHANGES_REQUESTED** [review]

File: books.md (hunk 3 above)
Comment (HasnaatCodes) [reply books.md:11]: I don't agree with this suggestion

**Pull request merged**